config_default.json

```diff
@@ -15,6 +15,10 @@
 		"port": 8080
 	},
 	"variables": {},
-	"korrektoren": {},
+	"korrektoren": {
+		"username": "alexander",
+		"passhash": "$pbkdf2-sha256$29000$yHkvRUiJMSZEyNnb.18rhQ$RIpwWd5GZfn/kEAcNVOEqoAprB.2j1beaAQUzWe2nVw"
```

Comment:
Umm, most certainly you did not want to publish your password (hash)!
`config_default.json` should not be changed. Instead, create a file `config.json`. The key in the dictionary should be the username (and not `username` or `passhash`) anyways.